src/api/registration/registration.controller.js

```diff
@@ -61,6 +60,25 @@ const createNewRegistration = async (
     hygieneCouncilCode
   );
 
+  const completeCacheRecord = Object.assign(
+    {},
+    {
+      "fsa-rn": postRegistrationMetadata["fsa-rn"],
+      reg_submission_date: postRegistrationMetadata.reg_submission_date
+    },
+    registration,
+    lcContactConfig,
+    {
+      completed: {
+        tascomi: undefined,
```

Comment:
At this point, we don't even know if we need a property for tascomi, so might be better to exclude it and add it later

Reply:
removed it, it will only get added if the tascomi get's called with a true or false depending on success